azure-pipelines.yml

```diff
@@ -1090,4 +1176,21 @@ stages:
     #     ApiPath: $(RecruitOuterApiPath)
     #     ApiBaseUrl: $(RecruitOuterApiBaseUrl)
     #     ProductId: $(RecruitProductId)
-    #     ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
\ No newline at end of file
+    #     ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
+    
+    #  - template: pipeline-templates/job/deploy-api.yml
+    #    parameters:
+    #      DeploymentName: Deploy_Campaign
+    #      DeploymentCondition: and(succeeded(), eq(${{ parameters.Campaign }}, true))
+    #      DeploymentApprovals: MO
+    #      ServiceConnection: SFA-DAS-ModelOffice-ARM
```

Comment:
Needs to be SFA-ASM-ModelOffice-ARM. Could you also fix this on the MO recruit deployment as well?

Reply:
👍🏻